src/components/message/message.twig

```diff
@@ -0,0 +1,23 @@
+{# 
+  @TODO
+  1. Generate an example of a message with a decorative icon.
+  #}
+
+{% set tagName = tagName | default('div') %}
+
+<{{tagName}} aria-describedby="MessageInfo" {% if not stopAnnouncements %} aria-live="polite"{% endif %} class="c-message{% if class %} {{class}}{% endif
+%}" tabindex=”0”{% if isWarning %} role="alert"{% else %} role="status"{% endif
+%}>
+  <p id="MessageInfo">
```

Comment:
Including an ID in markup by default prevents the pattern from being used more than once in a page.